config/config.go

```diff
@@ -179,3 +188,18 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultIMUD
 
 	return optionalConfigParams, nil
 }
+
+func validateModes(
+	optionalConfigParams OptionalConfigParams,
+	imuEnabled bool,
+) error {
+	localizationMode := optionalConfigParams.LidarDataRateMsec == 0
+	offlineMode := !optionalConfigParams.EnableMapping && optionalConfigParams.ExistingMap != ""
```

Comment:
Similarly here, should this be setting localizationMode?

Reply:
yes got them backwards fixing!